provider/lpmessage/sender.go

```diff
@@ -350,7 +350,10 @@ func (s *Sender) Send(ctx context.Context, msg *types.Message, mss *api.MessageS
 		var sigCidStr, sendError string
 		var sendSuccess *bool
 
-		err = s.db.QueryRow(ctx, `select signed_cid, send_success, send_error from message_sends where send_task_id = $1`, taskAdder).Scan(&sigCidStr, &sendSuccess, &sendError)
+		err = s.db.QueryRow(ctx, `
```

Comment:
oh, we should be persisting the taskID in the call to `taskAdder` a few lines above - this is the ID of the message send task (which is separate from the recover/submit task IDs)